srcpkgs/python3-binaryornot/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-binaryornot'
+pkgname=python3-binaryornot
+_pkgname=binaryornot
```

Comment:
remove this variable and replace where it's used with `binaryornot` (similar change should also be done in other packages)